mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/metrics/cgroups/Cgroup.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.worker.metrics.cgroups;
+
+import java.io.IOException;
+import java.util.Map;
+
+interface Cgroup {
```

Comment:
where is this used? I cannot seem to find the ref to use it in main path. Can you maybe add some docs to this interface to give how this is wired into container?

Reply:
Look into the implementations of CpuSubsystemProcess or MemorySubsystemProcess. It is used there. I'll also add some documentation to the classes added.